playbooks/common/openshift-cluster/upgrades/upgrade_control_plane.yml

```diff
@@ -29,6 +29,7 @@
 
 - name: Backup etcd
   include: ./etcd/backup.yml
+  when: openshift_upgrade_skip_etcd_backup is not defined or not openshift_upgrade_skip_etcd_backup | bool
```

Comment:
Is this part of some other change or just additional control you added while debugging this?
I think we've decided that we prefer the following rather than checking for is not defined
`when: openshift_upgrade_skip_etcd_backup | default(true) | bool`